src/views/Verbose.vue

```diff
@@ -71,6 +71,41 @@
             v-model="filter"
             :placeholder="$t('verbose.filterPlaceholder')"
           >
+          <table>
+            <tr>
+              <td>
+                <div
+                  :class="{ 'exclude-result': true, 'selected': isExcluded('true') }"
+                  @click="excludeResult('true')"
+                >
+                  <span></span>
+                </div>
+              </td>
+              <td>{{ $t('verbose.exclude') }} <code :class="'true'">true</code></td>
```

Comment:
When attributes are strings, you don't need to bind them like `:class="'true'"`, you can just write `class="true"`

Reply:
Not sure how I missed that, thanks 😄